File: error-handling-spring-boot-starter/src/main/java/com/ocean/bank/error/handling/dto/ExceptionDto.java
package com.ocean.bank.error.handling.dto;

import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

@AllArgsConstructor
@Data
@NoArgsConstructor
public class ExceptionDto {
    private String time;
    private String message;
    private String cause;
    private Integer status;
}

File: business-services/account-api/src/main/kotlin/com/ocean/bank/account/controller/dto/ExceptionDto.kt
package com.ocean.bank.account.controller.dto

import com.fasterxml.jackson.annotation.JsonInclude
import java.time.LocalDateTime

data class ExceptionDto(
    val time: String,
    val message: String?,
    @JsonInclude(JsonInclude.Include.NON_NULL)
    val cause: String?,
    val status: Int,
)
